pkg/storage/export/export/export.go

```diff
@@ -1342,15 +1342,8 @@ func (ctrl *VMExportController) expandVirtualMachine(vm *virtv1.VirtualMachine)
 
 func (ctrl *VMExportController) updateHttpSourceDataVolumeTemplate(vm *virtv1.VirtualMachine) *virtv1.VirtualMachine {
 	for _, volume := range vm.Spec.Template.Spec.Volumes {
-		volumeName := ""
-		if volume.DataVolume != nil {
-			volumeName = volume.DataVolume.Name
-		}
-		if volume.PersistentVolumeClaim != nil {
-			volumeName = volume.PersistentVolumeClaim.ClaimName
-		}
-		if volumeName != "" {
-			vm.Spec.DataVolumeTemplates = ctrl.replaceUrlDVTemplate(volumeName, vm.Namespace, vm.Spec.DataVolumeTemplates)
+		if types.PVCNameFromVirtVolume(&volume) != "" {
```

Comment:
Please, use here a variable for the name. Conceptually, this isn't wrong, but the variable avoids to call the function twice